python/labbox_ephys/extractors/bandpass_filter.py

```diff
@@ -1,11 +1,24 @@
+import spikeextractors as se
```

Comment:
The bandpass filter (preprocessing) was inefficient, so I made some adjustments.